### DISCUSSION TITLE
How to localize the README file?

### DISCUSSION THREAD

**Discussion by zenglian** [community/community#22310] [New to GitHub]
<p>The expected behavior is:</p>
<p>Show different language according to browser locale.</p>
<p>e.g.,</p>
<p>User see contents of README_zh.md when the browser’s locale is zh; </p>
<p>User see contents of README_fr.md when the browser’s locale is fr.</p>
<p>Does github support this, or is there any plan to support it?</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/zenglian">@zenglian</a>,</p>
<p>This is not possible to do right now, but you can do something like this:</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181070150-97fc56b9-961a-4305-9f85-b32f86b5c9af.png" alt="or.png" data-base62-sha1="p1WBuGWcq14LX2Kw0k3394Olm4l" width="287" height="174"></p>
<p>where “English” link to a readme_en.md inside a directory where you put all the readme.</p>
<p>This is the solution I adopted for my project.</p>
<p>-Gabriele-</p>
<blockquote>
<p><strong>Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great content!</strong></p>
</blockquote>

**Reply by clins1994** [reply]
@Wabri genius
